### ISSUE TITLE
Cannot add more of a custom gear in career mode

### ISSUE THREAD

**Issue by Lochabar** [chummer5a/chummer5a#1261]
### Environment
Chummer Version: 5.188.15
Environment: Windows 7)
Runtime: .NET 4.6.1

### Expected behaviour
Be able to "buy more" custom gear.

### Actual behaviour
On the gear window, working with a custom item, when the '+' button is used to buy more gear, the resulting window shows the wrong purchase amount, and wrong page reference.  Clicking "OK" does nothing.

### Steps to reproduce the behaviour
Create a custom item, try to add more of the item.

### Link to affected character file
The current gear definition:
https://www.dropbox.com/s/xpyr1128vtdoztq/Gear%20definition.png?dl=0

The "+" screen
https://www.dropbox.com/s/h5i7dy24isjb4bi/Gear%20to%20add.png?dl=0

**Comment by chummer5a** [comment]
When you say custom item, do you mean an item that was created from the Custom Item object, or one that was created via XML?

**Comment by chummer5a** [comment]
Makes sense, quantity changes are done by looking for the object in the XML files so you can do Markup and such. 'Best' way to manage this will probably have to be creating a proper object that gets injected into a custom XML file.
